mappings/net/minecraft/server/network/PendingConnectionManager.mapping

```diff
@@ -0,0 +1,11 @@
+CLASS net/minecraft/class_9 net/minecraft/server/network/PendingConnectionManager
+	FIELD field_28 logger Ljava/util/logging/Logger;
+	FIELD field_29 listening Z
+	FIELD field_30 minecraftServer Lnet/minecraft/server/MinecraftServer;
+	FIELD field_31 socket Ljava/net/ServerSocket;
+	FIELD field_32 listenThread Ljava/lang/Thread;
```

Comment:
should this be listeningThread?

Reply:
the title is "listen thread", i can change it if you want

Reply:
wdym by "the title"

Reply:
Name of the thread

Reply:
did you get this from in-code strings?

Reply:
yes

Reply:
listeningThread sounds better imo, though that's just my opinion